flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -95,38 +82,31 @@
 
 /** IT cases for all cassandra sinks. */
 @SuppressWarnings("serial")
-@Category(FailsOnJava11.class)
-// this test is known to be unstable, but the exact cause is unknown
-@RetryOnException(times = 2, exception = NoHostAvailableException.class)
 public class CassandraConnectorITCase
         extends WriteAheadSinkTestBase<
                 Tuple3<String, Integer, Integer>,
                 CassandraTupleWriteAheadSink<Tuple3<String, Integer, Integer>>> {
 
-    private static final Logger LOG = LoggerFactory.getLogger(CassandraConnectorITCase.class);
-
-    @Rule public final RetryRule retryRule = new RetryRule();
-
-    private static final boolean EMBEDDED = true;
-
-    private static EmbeddedCassandraService cassandra;
+    private static final String IMAGE_TAG = "3.0";
 
-    private static final String HOST = "127.0.0.1";
+    @ClassRule public static CassandraContainer cassandra = createCassandraContainer(IMAGE_TAG);
```

Comment:
can be final

Reply:
Thanks. I put final on the other fields but I missed that one